qemu/tests/ksm_base.py

```diff
@@ -102,10 +101,10 @@ def _execute_allocator(command, vm, session, timeout):
 
     query_cmd = re.sub("QEMU_PID", str(vm.process.get_pid()), query_cmd)
 
-    sharing_page_0 = decode_to_text(process.system_output(query_cmd,
-                                                          verbose=False,
-                                                          ignore_status=True,
-                                                          shell=True))
+    sharing_page_0 = process.system_output(query_cmd,
```

Comment:
@luckyh I've chosen to use the `process.run(...).stdout_text` pattern instead in the Avocado-VT PR https://github.com/avocado-framework/avocado-vt/pull/2450. The style you chose is not wrong, but maybe we want to attempt a similar style.

Reply:
no problem, have updated, thanks!